source/includes/table-permissions-self-hosted.rst

```diff
@@ -4,47 +4,58 @@
 
 .. list-table::
    :header-rows: 1
+   :stub-columns: 1
+   :widths: 15 20 65
 
    * - Sync Type
-     - Target
+     - Cluster
```

Comment:
**Suggestion**

Per the comment on the Jira, "For reversing the user will need the dbAdmin role on the original destination cluster. That's because it needs to drop a database on that cluster. Note that this role is not needed for write-blocking, so we should find a way to indicate that." You could add a note column for that additional level of detail, I think we are trying to get away from footnotes [1] but that might be another option.

Reply:
Fixed